exporters/stdout/metric.go

```diff
@@ -123,6 +148,29 @@ func (e *metricExporter) Export(_ context.Context, checkpointSet metric.Checkpoi
 					}
 				}
 			}
+
+			if hist, ok := agg.(aggregation.Histogram); ok {
+				val, err := hist.Histogram()
+				if err != nil {
+					return err
+				}
+				buckets := make([]bucket, 0, len(val.Boundaries))
+				lowerBound := float64(-math.MaxFloat64)
```

Comment:
Shouldn't this be negative infinity? Or is there a limitation here?

```suggestion
				lowerBound := math.Inf(-1)
```